Update core-autodeploy.sh

zonshy opened the pull request:
error: 1.10 /opt/serviced/var/backups filesystem check  failed. 
because it cannot detected the backup mount filesystem

zonshy committed:
Update core-autodeploy.sh

jangaraj commented:
@zonshy Could you provide also some comment about your patch and problem please?

zonshy committed:
Update core-autodeploy.sh

it maybe cause "/opt/serviced/var/backups filesystem was not detected". the Escapes is invailed

jangaraj commented:
@zonshy :+1:
